app/views/developers/show.html.erb

```diff
@@ -57,6 +57,17 @@
                   <% end %>
                 </div>
               <% else %>
+                  <% if current_user.company && current_user.company.favourites.include?(@developer) %>
+                    <%= link_to developer_favourite_path(current_user.company.developer_favourites.find_by(developer: @developer)), method: :delete, class: 'btn btn-flat', style: 'font-size:0.88rem' do %>
+                      <i class="fas fa-ban fa-lg"></i> Remove from favourites
+                    <% end %>
+                    <% else %>
+                    <%= link_to developer_developer_favourites_path(@developer), method: :post, class: "btn btn-flat favourite" do %>
+                    <i class="far fa-heart fa-lg"></i> Add to favourites</a>
+                    <% end %>
+                  <% end %>
+                  <a class="btn btn-flat message w-100" href="#"><i class="fab fa-telegram-plane fa-lg"></i> Send message</a>
+
                 <div class="d-flex flex-column align-items-start my-2 w-100">
                   <% if @developer.website.present?  %>
                     <%= link_to "#{"https://" if !@developer.linkedin.include?('http')}#{@developer.website}", class: "btn btn-flat w-100 mb-3" do %>
```

Comment:
Down at line 77, "add to favourites is in there again, as is "Send message". Please remove these duplicates.